lib/src/scene_runner/components/material.rs

```diff
@@ -132,14 +137,19 @@ pub fn update_material(scene: &mut Scene, crdt_state: &mut SceneCrdtState) {
                             .0
                             .clone()
                             .multiply(pbr.emissive_intensity.0);
-                        godot_material.set_emission(emission.to_godot());
-                        godot_material.set_albedo(pbr.albedo_color.0.to_godot());
+
+                        // In the Mobile renderer, HDR will be capped at 2.0 so we'll have to reduce the energy multiplier to be able to see fluctuations in energy
+                        godot_material.set_emission_energy_multiplier(0.1);
+
+                        // In the same way, godot uses sRGB instead of linear colors.
+                        godot_material.set_emission(emission.to_godot().linear_to_srgb());
```

Comment:
Great. Remember this only happens on `Materials` that the Scene Creators overrides.

The Materials from a GLTF are not affected.